x-pack/plugins/cases/server/services/attachments/index.ts

```diff
@@ -372,6 +461,34 @@ export class AttachmentService {
     );
   }
 
+  public async find({
```

Comment:
@jonathan-buttner I moved this function to the attachment service so it can be used by the CaseService to get all comments.